front/src/modules/trainschedule/components/Timetable/PacedTrain/OccurrenceItem.tsx

```diff
@@ -55,37 +57,66 @@ const OccurrenceItem = ({
       })}
       role="button"
       tabIndex={0}
-      onClick={() => selectOccurrence(occurrence.id)}
+      onClick={() => {
+        if (isOccurrence(occurrence.id)) selectOccurrence(occurrence.id);
```

Comment:
I think you can remove the test here
```suggestion
       selectOccurrence(occurrence.id);
```

Reply:
Ok, it's needed actually :white_check_mark: